doc/esmvalcore/preprocessor.rst

```diff
@@ -218,6 +218,20 @@ extract the levels and vertically regrid onto the vertical levels of
           # levels: {dataset: ERA-Interim, coordinate: air_pressure}
           scheme: linear_horizontal_extrapolate_vertical
 
+By default, vertical interpolation is performed in the dimension coordinate of
+the z axis. If you want to use another coordinate (for example, ``air_pressure``'
+in variables that are provided in model levels and not pressure levels) you can
```

Comment:
I think `explicitly declaring the z-axis coordinate name` is the key phrase here